RELEASE_PLAN.md

```diff
@@ -1,49 +1,49 @@
-* In case of unexpected major updates of important dependencies like Vue, Webpack, and so on, major versions might be released *earlier* than planned.
+- If there are no breaking changes waiting for a release, no new major version will be published. Instead, another minor one will be released
+- In case of unexpected major updates of important dependencies like Vue, Webpack, and so on, major versions might be released _earlier_ than planned
 
 The goal is to provide a **migration guide** for each major version as well, as escape hatches, so existing code
 won't "just break".
 
 ### Minor versions (2.1 -> 2.2)
 
-The release cycle for Nuxt.js minor versions is roughly **4 weeks**. 
+The release cycle for Nuxt.js minor versions is roughly **4 weeks**.
 
-Three of the four weeks will be used for actual **feature implementations** while the last week will be used for 
+Three of the four weeks will be used for actual **feature implementations** while the last week will be used for
 **testing, fixing bugs and thorough audits**.
 
-That also means a *feature freeze* for the next minor version after these three weeks.
-Features that aren't ready will be moved to the next cycle. "Waiting" for features 
+That also means a _feature freeze_ for the next minor version after these three weeks.
+Features that aren't ready will be moved to the next cycle. "Waiting" for features
 (for a longer time) will be avoided as good as possible to keep releases lean, concise, predictable and digestible.
 
 ### Patch releases (2.2.3 -> 2.2.4)
 
-The last patch releases were mostly *bundled* fixes or single *hotfixes*. 
-In the future, fixes will be released **as soon as possible** after the actual PR/commit so people won't have to switch to `nuxt-edge` for bugfixes. This should improve the stability of Nuxt
+The last patch releases were mostly _bundled_ fixes or single _hotfixes_.
+In the future, fixes will be released **as soon as possible** after the actual PR/commit so people won't have to switch to `nuxt-edge` for bugfixes. This should improve the stability of Nuxt.
 
 Fixes can or will include:
 
-* Updates of dependencies (for various reasons, like a "faulty/buggy" dependency or an newer versions that works better with the Nuxt.js code)
-* Fixes for our code
+- Updates of dependencies (for various reasons, like a "faulty/buggy" dependency or an newer versions that works better with the Nuxt.js code)
+- Fixes for our code
 
 Bugfixes for upcoming features won't be ported of course.
 
 ### Edge Release Channel
 
-After experimenting with `nuxt-edge` releases in the last time, the decisiion to do **nightly releases** for now instead of 
-releasing a version after *each commit* was made.
+After experimenting with `nuxt-edge` releases in the last time, the decisiion to do **nightly releases** for now instead of
+releasing a version after _each commit_ was made.
```

Comment:
Any reason why there is a switch from `*` syntax to `_` syntax?

Reply:
Nice point! I think it is just like a preference. I usually use `_` but the spec mentions both equally:

https://spec.commonmark.org/0.29/#emphasis-and-strong-emphasis

Reply:
Like @pi0 said, Both are the same thing! But I like using _ instead of * because:
- It's easy to distinguish * vs **(bold).
- VSCode automatically format the code became _ instead of * => Easiest way to format markdown!
@manniL

Reply:
Let's get the boss answer :D 

https://twitter.com/_pi0_/status/1198659461533110274

(anyway we can don't block this PR for it)

Reply:
I don't mind using `_`. Just wondered what the intention behind it was. Didn't know that VSCode prefers _ over * :relaxed:

Reply:
If you know someone in vscode developers team => Ask them for it 🥳. I also want to know why!